src/site/content/en/blog/building-an-adaptive-favicon/index.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Building an adaptive favicon
+subhead: A foundational overview of how to build an adaptive favicon.
+authors:
+  - adamargyle
+description: A foundational overview of how to build an adaptive favicon.
+date: 2022-02-09
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/UGZpusKrxwNKE15Cm2lV.png
+alt: Chrome, Firefox and Safari shown, in light and dark themes, showcasing the favicon.
+thumbnail: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/on4LayLT6bZOGcfyPIYa.png
+tags:
+  - blog
+  - css
+  - svg
+---
+
+In this post I want to share thinking on how to build an adaptive favicon with
+SVG. Try the [demo](https://gui-challenges.web.app/svg-favicon/dist/).
+
+<figure>
+  {% Video
+    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/SfYwSCUREc4XrS0ZAiZN.mp4",
+    autoplay="true",
+    loop="true",
+    muted="true"
+  %}
+  <figcaption>
+    Browser tabs shown adapting to MacOS system light and dark theme changes. 
+    <a href="https://gui-challenges.web.app/svg-favicon/dist/">Try the Demo</a>
+  </figcaption>
+</figure>
+
+If you prefer video, here's a YouTube version of this post:
+
+{% YouTube 'R75ZVW4LW5o' %}
+
+## Overview
+
+A custom [favicon](https://developer.mozilla.org/docs/Glossary/Favicon) is
+a great way to polish a web project. It displays on desktop browser tabs, and
+also inside "save for later" readers, other blog posts linking to your site, and
+more. Traditionally this has been done with the `.ico` file type, but recently
+browsers have allowed use of
+[SVG](https://developer.mozilla.org/docs/Web/SVG), a vector format. Using
+[progressive
+enhancement](https://developer.mozilla.org/docs/Glossary/Progressive_Enhancement)
+you can  serve well supported `.ico` favicons, and upgrade to an `.svg` if
+available.
+
+{% BrowserCompat 'api.linkIconSVG' %}
```

Comment:
i can find the caniuse for SVG favicons but struggled to find the api key for it for use here..

Reply:
is there a good way to find the ID from [here](https://caniuse.com/link-icon-svg)?

Reply:
I replied about this in chat but it isn't in BCD data.